test/testing/evaluators.spec.ts

```diff
@@ -230,26 +191,6 @@ describe('OOB Evaluators', () => {
       fn: ({ testCase }: { testCase: MyTestCase }) => testCase.input,
     });
 
-    expectNumRequests(5);
```

Comment:
the only assertions we really need in these tests are if the evals were sent correctly